lib/xlsx/xform/base-xform.js

```diff
@@ -64,9 +64,7 @@ class BaseXform {
         } else if (eventType === 'text') {
           this.parseText(value);
         } else if (eventType === 'closetag') {
-          if (!this.parseClose(value.name)) {
-            return this.model;
-          }
+          if (!this.parseClose(value.name)) break;
```

Comment:
which issue is it fix?

Reply:
"Fixes a stall (unreported?) in the BaseXForm parser - if closetag was encountered it would return, and the sax parser would fail to flush and just pause forever. The correct behavior should be to break so that it flushes the parser correctly."